### PR TITLE
Implement "stream broadcast round pgn" endpoint

### LINKED ISSUE #36: Implement "stream broadcast round pgn" endpoint
Hey i'm working on a project and if you guys manage to implement this /api/stream/broadcast/round/{broadcastRoundId}.pgn endpoint

### PR DESCRIPTION
**Pull request by devjiwonchoi**
Fixes #36

### CODE REVIEW

**Comment by Virinas-code** [comment]
Maybe rename `broadcast_round_id` to `round_id`?

**Comment by devjiwonchoi** [comment]
@Virinas-code Since other functions use `broadcast_round_id`, I think keeping it consistent would be better.

**Comment by Virinas-code** [comment]
> @Virinas-code Since other functions use `broadcast_round_id`, I think keeping it consistent would be better.

Probably. I didn't check out the rest of the code :sweat_smile:.

File: README.rst [hunk 1]
```diff
@@ -104,7 +104,8 @@ Most of the API is available:
     client.broadcasts.get_round
     client.broadcasts.update_round
     client.broadcasts.get_round_pgns
-    client.broadcasts.get_pgns
```
Comment (icp1994) [inline README.rst:107]: this function still exists so why remove it?

Comment (devjiwonchoi) [reply README.rst:107]: Oh, my bad.

**Review by icp1994 — APPROVED** [review]

**Comment by kraktus** [comment]
Thanks! Apologies for the delay

**Pull request merged**